pkg/server/plugin/zmq/broker.go

```diff
@@ -48,23 +50,45 @@ const (
 	// It is an addition to original PPP to shorten the time needed for
 	// broker to detect a normal shutdown of worker.
 	Shutdown = "\003"
+	Request = "REQ"
+	Response = "RES"
 )
 
-// parcel is used to multiplex the chan with zmq worker
-type parcel struct {
-	respChan chan []byte
-	frame    []byte
-	retry    int
+// Parcel is used to multiplex the chan with zmq worker
+type Parcel struct {
```

Comment:
Why make it public? I didn't see it used outside of the `zmq` package, and if there are such case, we probably have other problems.